tests/test_crypto.py

```diff
@@ -1230,8 +1230,8 @@ def test_check_pr_897(self):
         """
         `PKey.check` raises `OpenSSL.crypto.Error` if provided with broken key
```

Comment:
Can you update this docstring to say that either `load_privatekey` or `check` raises

Reply:
Done. Also I missed that we actually ran into this in two different places, so updated the other test too.